test/library/standard/List/initEquals/listInitEqualsArrayBug.bad

```diff
@@ -0,0 +1,3 @@
+listInitEqualsArrayBug.chpl:5: warning: list([domain(1,int(64),one)] int(64),false)
+listInitEqualsArrayBug.chpl:9: warning: list([domain(1,int(64),one)] int(64),false)
+timedexec: target program died with signal 11, without coredump
```

Comment:
This might have different behavior on Macs versus Linux, but I think it's worth trying to have a .bad file and seeing what happens

Reply:
This passed as a future on my Mac and timed-out on Linux. I can just remove the `.bad` file to prevent it failing in nightly

Reply:
Sounds good, I think this is good to go once that adjustment happens